lib/StockTransformation.ts

```diff
@@ -0,0 +1,10 @@
+export interface StockQueueEntry {
+    product_id: string
+    variant_id: string | null
+    location_id: string
+    count: number
+}
+
+export interface StockTransformation {
+    transformToStockImportData(input: any, configuration: any, callback: (stockQueueElement: any) => Promise<void>): Promise<void> 
```

Comment:
Kunne interfacet ikke tage en StockQueueEntry istedet for en any?

Reply:
Som`input`? Det er noget der kommer fra Shopify og skulle gerne opfylde InventoryLevelUpdate interfacet

Reply:
Ahh du mener måske til `stockQueueElement`?